src/helpers.ts

```diff
@@ -79,3 +79,31 @@ export class CinnamonError extends AdgoError {
         this.name = 'CinnamonError';
     }
 }
+
+export const sleep = (time: number) =>
+    new Promise(resolve => setTimeout(() => resolve(), time));
+
+export type AugmentedRequired<
+    T extends object,
+    K extends keyof T = keyof T
+> = Omit<T, K> & Required<Pick<T, K>>;
+
+export const NON_RETRYABLE_ERROR_CODES = [
```

Comment:
Is this something what would make more sense being defined in the graphql-errors lib? I just don't like the idea that if we add an error code to the lib, then it will suddenly be whitelisted for retry by default.

Reply:
fixed